mediatek: comfast cf-e393ax: Add missing WAN port label and correct image defines

db260179 opened the pull request:
for this board

WAN port label was missing from the dts config so on intial config bootup would only set the lan port correctly.

Removing invalid image settings - DEVICE_DTC_FLAGS, DEVICE_DTS_LOADADDR and KERNEL_LOADADDR as had noticed on factory install kernel would be allocated incorrectly

obsy commented on target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts:
```diff
@@ -216,6 +216,11 @@
 		};
 
+		port@1 {
+			reg = <1>;
```
So this should be changed too: https://github.com/openwrt/openwrt/blob/main/target/linux/mediatek/filogic/base-files/etc/board.d/02_network#L57

db260179 committed:
mediatek: comfast cf-e393ax: Add missing WAN port label and correct image defines

for this board

WAN port label was missing from the dts config so on intial config bootup would only set the lan port correctly.

Removing invalid image settings - DEVICE_DTC_FLAGS, DEVICE_DTS_LOADADDR and KERNEL_LOADADDR as had noticed on factory install kernel would be allocated incorrectly

Signed-off-by: David Bentham <db260179@gmail.com>

hauke commented on target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts:
```diff
@@ -216,6 +216,11 @@
 
+		port@1 {
+			reg = <1>;
+			label = "eth1";
```
If port 1 on the switch is eth1 now, is `gmac1` still used? If it is not used you should remove it from the device tree.
